NAMESPACE

```diff
@@ -31,6 +31,7 @@ export(mmrm)
 export(mmrm_control)
 export(refit_multiple_optimizers)
 import(checkmate)
+import(utils)
```

Comment:
ahm no, we don't want to import the whole package. imports should be as minimal as possible. especially here we just use 5 functions so far from the pkg.